knimin/templates/ag_pulldown.html

```diff
@@ -71,13 +80,33 @@ <h3 style="color:red">Pulldown Processing, please wait for file download. It may
 <form enctype="multipart/form-data" action="/ag_pulldown/" name="agForm" id="agForm" method="post">
 <p>Upload qiime mapping file, or other file with first line header and one barcode per line</p>
 <p>Barcodes File <input type="file" name="barcodes" id="barcodes"/></p>
-<p>External Survey
+<p>American Gut Surveys:
+  <div>Select which American Gut surveys shall be included in the pulldown archive. <br/>Add 'all in one file' if you also want to have one addition file<br/>that contains all available columns from all American Gut surveys.</div>
+  <select name="agsurveys" id="agsurveys" multiple />
+  {% for s in agsurveys %}
+  {% if s[2] == True %}
+  <option value='{{s[0]}}' selected>{{s[1]}}</option>
+  {% else %}
+  <option value='{{s[0]}}'>{{s[1]}}</option>
+  {% end %}
+  {% end %}
+  </select>
+</p>
+	
+</p>
+<p>External Survey:
+  <div>Select which external surveys should be included in the pulldown archive.</div>
   <select name="external" id="external" multiple />
   {% for s in surveys %}
   <option value='{{s}}'>{{s}}</option>
   {% end %}
   </select>
 </p>
+{% if merged == 'True' %}
```

Comment:
yes it is, since 'merged' holds the content of 'value' of the checkbox which I defined as the string 'True'.